fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
In theory, this check could be a little more robust — handling both the absence of Link headers or simply not having any of the Fedora Link headers that are checked in the next section.  But I would be fine with just updating the comment about how this relies on the magic /fcr:metadata suffix.